lib/gui/os/window-progress.js

```diff
@@ -38,12 +39,16 @@ exports.currentWindow = electron.remote.getCurrentWindow();
  * Show progress inline in operating system task bar
  *
  * @param {Number} percentage - percentage
+ * @param {String} status - flashing/validating status
  *
  * @example
  * windowProgress.set(85);
  */
-exports.set = (percentage) => {
+exports.set = (percentage, status) => {
```

Comment:
Can we decouple this module from Etcher's use case by naming it accept a generic title, and calculate what that title should actually be from outside?